build-system/.eslintrc.js

```diff
@@ -48,6 +48,6 @@ module.exports = {
     'local/no-module-exports': 0,
     'local/no-rest': 0,
     'local/no-spread': 0,
-    'require-jsdoc': 0,
+    'require-jsdoc': 1,
```

Comment:
I strongly suggest that you make this change in a follow-up PR - when your PR edits so many files the odds of there being another change that edits one of the same files increases significantly, and it's quite possible that someone will add a new function or change an existing one without the required JSDoc - which would cause the two merged to result in a codebase that breaks the build (and on a mere technicality no less)

If you first merge the changes, then wait a ~day (or at least couple hours), and then make the change, you are much more likely to avoid this situation

Reply:
That is a great point

Reply:
+1. Thanks for this substantial cleanup @rileyajones, and thanks for pointing this out, @danielrozenberg!